pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -812,31 +814,41 @@ def show_help_instancename():
 """)
 
 
-def get_instancename(context, instancename, options, ns_names=None):
+def get_instancename(context, instancename, options, default_all_ns=False):
     """
     Common function to construct a CIMInstanceName object from the
     INSTANCENAME argument and the --key and --namespace options specified
     in the command line.
 
-    The keybindings of the returned CIM instance path must be specified in
+    If the ns_names option is a list, the request is to return instance
```

Comment:
1. You removed the `ns_names` parameter.
2. It is a parameter, not an option.
3. It is boolean and not a list.

Reply:
Fixed. Removed reference to ns_names.

Reply:
DONE Removed from parameter list and elsewhere in documentation